app/src/test/java/org/mozilla/fenix/library/history/HistoryControllerTest.kt

```diff
@@ -189,4 +202,13 @@ class HistoryControllerTest {
         assertEquals(historyItem.title, (directions.captured.arguments["data"] as Array<ShareData>)[0].title)
         assertEquals(historyItem.url, (directions.captured.arguments["data"] as Array<ShareData>)[0].url)
     }
+
+    @Test
+    fun onRequestSync() {
+        controller.handleRequestSync()
+
+        verify {
+            syncHistory.invoke(any())
```

Comment:
I think we should also verify here that the right actions are emitted.
`StartSync` is easy to be verified, but verifying that syncHistory gets passed the `FinishSync` callback is a bit trickier.. maybe we could return that store.dispatch call from a method and then in the test verify if 
- `store.dispatch(HistoryFragmentAction.StartSync)` was called
- `store.dispatch(any())` was called 2 times
- the method returning `store.dispatch(HistoryFragmentAction.FinishSync)` was called ?